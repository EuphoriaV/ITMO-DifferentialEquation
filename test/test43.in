2.78
(((4.77 + x) * x) * ((x + (x * x)) * ((6.68 + x) * (4.07 - x))))
3.9
2.72