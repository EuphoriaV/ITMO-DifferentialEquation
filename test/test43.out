-0.0227658